pkg/iostreams/iostreams.go

```diff
@@ -111,12 +111,8 @@ func (s *IOStreams) StartPager() error {
 			pagerEnv = append(pagerEnv[0:i], pagerEnv[i+1:]...)
 		}
 	}
-	if _, ok := os.LookupEnv("LESS"); !ok {
-		pagerEnv = append(pagerEnv, "LESS=FRX")
-	}
-	if _, ok := os.LookupEnv("LV"); !ok {
-		pagerEnv = append(pagerEnv, "LV=-c")
-	}
+
+	pagerEnv = append(pagerEnv, "LESS=FrX", "LV=-c")
```

Comment:
Maybe we should only override the user's Less config only when they set FORCE_HYPERLINK=1 ??

Reply:
Good point.